src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRulesLoader.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.strimzi.kafka.bridge.mqtt.core.MqttKafkaMapper;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Helper class to Load the rules from the configuration file
+ */
+public class MappingRulesLoader {
+
+    private static final MappingRulesLoader instance = new MappingRulesLoader();
```

Comment:
Not sure I understand we we create a new instance here in this class, why not just rely on the constructor?

Reply:
I have answered [here](https://github.com/strimzi/strimzi-mqtt-bridge/pull/8#discussion_r1240604882)

Reply:
```suggestion
    private static final MappingRulesLoader INSTANCE = new MappingRulesLoader();
```